addons/website/models/website.py

```diff
@@ -154,6 +154,19 @@ def write(self, values):
             self.env['ir.qweb'].clear_caches()
         return result
 
+    @api.multi
+    def unlink(self):
+        # Do not delete invoices, delete what's strictly necessary
+        attachments_to_unlink = self.env['ir.attachment'].search([
```

Comment:
Everyone agree with this domain?
Any theme attachment, theme customization or assets should be removed on website delete.
Anything else?

Reply:
Ok for me. As discussed, maybe we can make this more robust by sharing some of the code which adds this ".custom."... but maybe we should just take profit of my original idea of adding special tags on those attachment now that the document team had the right to do it :)